src/Symfony/Component/Yaml/Exception/UnsupportedTagException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Yaml\Exception;
+
+/**
+ * @author Guilhem N. <egetick@gmail.com>
+ *
+ * @internal
```

Comment:
It should be part of the api in 4.0 but as it never ends in userland for now, making it internal allows us to change its name (or maybe just remove it in favor of a simple `ParseException`?).